### PR TITLE
Add Github actions so CI runs

### PR DESCRIPTION
**Pull request by simPod**

### CODE REVIEW

**Comment by cboden** [comment]
You're a saint @simPod! Do you know what I need to do in order for GitHub actions to run as part of the PR?

**Comment by WyriHaximus** [comment]
@cboden it needs to be merged before they are ran the first time.

**Comment by simPod** [comment]
@cboden we can merge and I'll fix whatever is left

**Comment by cboden** [comment]
@simPod Sounds good. Can you do 2 quick things before I merge please?

1. Target master branch instead of v0.5 (I'll rebase master onto v0.5 later); I think this is important enough to fix for the current supported version, as well as next. 
2. Remove code coverage

**Comment by simPod** [comment]
@cboden should be done

File: README.md [hunk 1]
```diff
@@ -81,3 +81,7 @@ class MyChat implements MessageComponentInterface {
     conn.onmessage = function(e) { console.log(e.data); };
     conn.onopen = function(e) { conn.send('Hello Me!'); };
 ```
+
+[GA Image]: https://github.com/ratchetphp/Ratchet/workflows/CI/badge.svg
```
Comment (cboden) [inline README.md:85]: Should this be .github/workflows?

Comment (simPod) [reply README.md:85]: Not sure what you mean but it links to the github badge url for workflow named `CI`. Above url is correct.

https://docs.github.com/en/actions/managing-workflow-runs/adding-a-workflow-status-badge

Comment (cboden) [reply README.md:85]: ah, my bad

**Comment by simPod** [comment]
I wonder whether old PHP versions fail or not 🙃

**Comment by simPod** [comment]
Looks like it works. Some things failed but I fixed those for v0.5 if I remember correctly.

**Pull request merged**